### PR TITLE
CIRC-6470 IRONdb native Whisper read support

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: content/irondb/getting-started/configuration.md [hunk 1]
```diff
@@ -785,6 +787,51 @@ requesting data at shorter intervals.
...
 Default: 1 minute
 
+#### graphite whisper
+
+The `whisper` entity configures [read access to Whisper database
+files](/irondb/integrations/graphite/#native-whisper-read-support). Each entity
+refers to the top of a directory hierarchy containing Whisper database files.
+This directory may exist on a local filesystem, or on a shared
+network-filesystem mountpoint. Any Whisper databases discovered in scanning
+this directory hierarchy will be indexed for searching and querying.
```
Comment (bdunavant) [inline content/irondb/getting-started/configuration.md:797]:
```suggestion
this directory hierarchy will be indexed for searching and querying by the `whisper_loader` tool.
```
I think we should at least mention the tool here.  If someone goes straight to this page and ignores the main content, they may think this happens automatically.

**Review by bdunavant — CHANGES_REQUESTED** [review]
We don't mention mounting the file system read-only anywhere.  While not specifically a requirement, it probably is in order to mount on multiple nodes, and I think you had special flags for how it was mounted for proper operation.   As such, should we have a section describing the right way to do those flags?

**Comment by esproul** [comment]
> We don't mention mounting the file system read-only anywhere. While not specifically a requirement, it probably is in order to mount on multiple nodes, and I think you had special flags for how it was mounted for proper operation. As such, should we have a section describing the right way to do those flags?

The specific flags could differ depending on the type of filesystem being mounted, but we can definitely mention read-only as the preferred option.

File: content/irondb/getting-started/configuration.md (hunk 1 above)
Comment (esproul) [reply content/irondb/getting-started/configuration.md]: Used different wording but same idea.

**Review by bdunavant — APPROVED** [review]
LGTM

**Pull request merged**